openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,22 @@
+[id='op-working-with-pipelines-using-developer-perspective']
+= Working with {op-title} using the Developer
+include::modules/openshift-pipelines-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: working-with-pipelines-using-developer-perspective
+toc::[]
+
+You can use the *Developer* perspective of the OpenShift web console to create CI/CD pipelines for your software delivery process while creating an application on OpenShift Container Platform.
+
+After you create pipelines, you can view and visually interact with your deployed pipelines.
+
+NOTE: The .NET Core runtime is not yet supported by the pipeline templates.
```

Comment:
Please comment this note out, this is part of the templates/create section in ODC which is still DP. Once we are TP, we will need to check and see if this limitation still exists.